### PR TITLE
chore(docker): update to rh ubi8 minimal base image

### PR DESCRIPTION
**Pull request by aslakknutsen**

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Interesting. I have a branch with that open for a while and never managed to get the e2e build running https://github.com/bartoszmajsak/istio-workspace/tree/ubi_image

File: build/DockerfileTest [hunk 1]
```diff
@@ -1,7 +1,4 @@
-FROM centos:7
-
-RUN adduser istio-workspace
-USER istio-workspace
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```
Comment (bartoszmajsak) [inline build/DockerfileTest]: Are we ok with moving target as a base image?

Comment (aslakknutsen) [reply build/DockerfileTest]: Well.. Not really, but it's technically not any more moving than the one we had, as it's version locked by name, not tag. centos:7 vs uib8:latest would be same same was my thinking.

Comment (bartoszmajsak) [reply build/DockerfileTest]: Can we lock it to latest `8.0-131` then?

Comment (aslakknutsen) [reply build/DockerfileTest]: 8.0?

Comment (bartoszmajsak) [reply build/DockerfileTest]: yeah `8.0` would do, even though == latest :)

Comment (bartoszmajsak) [reply build/DockerfileTest]:
```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0
```

File: build/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,3 @@
-FROM centos:7
-
-RUN adduser istio-workspace
-USER istio-workspace
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```
Comment (bartoszmajsak) [inline build/Dockerfile]:
```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0
```

**Pull request merged**